site/en/publications.md

```diff
@@ -8,103 +8,113 @@ lang-ref: publications
 ## Publications

```

Comment:
publications and presentations confused as in the german version

Reply:
fixed in 3afafc85